docs/reference/configuration/data_types.rst

```diff
@@ -589,20 +589,20 @@ multiple vectors:
 
 If you in the observation use the ``SUMMARY_OBSERVATION`` or
 ``HISTORY_OBSERVATION`` keyword to compare simulatons and observations for a
-particular summary vector *that vector is automatically added* to the ERT
-configuration.
+particular summary vector you need to add this vector after SUMMARY in the ERT
+configuration to have it plotted.
 
 If you use the keyword ``REFCASE`` to configure an Eclipse reference case you
 can use wildcard notation to all summary vectors matching a pattern, i.e. this:
 
 ::
 
    REFCASE eclipse/refcase/CASE
-   SUMMARY WWCT:* WGOR:*
+   SUMMARY WWCT*:* WGOR*:*
    SUMMARY F*
    SUMMARY G*:NORTH
 
-will load the ``WWCT`` and ``WGOR`` vectors for all wells, all field related
+will load the ``WWCT`` and ``WWCTH`` (``WGOR`` and ``WGORH``) vectors for all wells, all field related
```

Comment:
Why `WGOR` in parantheses? (and you may line wrap the paragraph after the edits)

Reply:
Nice suggestion. Thanks.